scripts/accept.ts

```diff
@@ -0,0 +1,77 @@
+import { PublicKey } from "@solana/web3.js";
+import { getAuthorityPDA, DEFAULT_MULTISIG_PROGRAM_ID } from "@sqds/sdk";
+import BN from "bn.js";
+import {
+  parseArgs,
+  createWallet,
+  initEscrowProgram,
+  parseMultisig,
+  executeOrPropose,
+} from "./helpers";
+
+const BPF_UPGRADABLE_LOADER = new PublicKey(
+  "BPFLoaderUpgradeab1e11111111111111111111111"
+);
+
+async function main() {
+  const args = parseArgs();
+  const wallet = await createWallet(args);
+  const { connection, program, programId } = initEscrowProgram(args, wallet);
+  const multisigAddress = parseMultisig(args);
+
+  const programToTransfer = new PublicKey(args.program);
+  const currentAuthority = new PublicKey(args.authority);
+
+  const programData = PublicKey.findProgramAddressSync(
+    [programToTransfer.toBuffer()],
+    BPF_UPGRADABLE_LOADER
+  )[0];
+
+  // In multisig mode, the vault becomes the new authority
+  let newAuthority: PublicKey;
+  if (multisigAddress) {
+    [newAuthority] = getAuthorityPDA(
+      multisigAddress,
+      new BN(1),
```

Comment:
I think you're supposed to use the authority index that's specified in the multisig account (you can get it by calling `getMultisig`)